Add option to not show menu with items

david-livadaru-3pillar opened the pull request:
Update the top menu the be optional.
Rename number of items to number of elements because this number means than we have 'something' the pagination.
Fix index validation to be performed using numberOfElements.
Hide isValid(index:) provided by Array.

david-livadaru-3pillar committed:
feature/optional-top-menu

Update the top menu the be optional.
Rename number of items to number of elements because this number means than we have 'something' the pagination.
Fix index validation to be performed using numberOfElements.
Hide isValid(index:) provided by Array.

david-livadaru-3pillar committed:
hotfix/view-life-cycle-functions-called-multiple-times

Provide public constructor for segment.
Provide public getter for current index.

david-livadaru-3pillar committed:
origin/feature/optional-top-menu

Remove hardcoded red color from menu view controller.

flaviu88 commented on Sources/AppearenceController.swift:
```diff
@@ -130,7 +126,7 @@ class AppearenceController {
         }
         
-        if !isValid(transition.toIndex) {
+        if !menuViewController.isValid(index: transition.toIndex) {
```
Why don't you use the isValid method from this class?

david-livadaru-3pillar replied:
Initially the function was not providing correct value. Now it's fixed

david-livadaru-3pillar committed:
feature/optional-top-menu

Change status bar color to white.
Set automaticallyAdjustsScrollViewInsets to false to view controller containing the menu.
Fix state change of button while providing geometry objects.
Provide customization for adjust the screen's scroll offset.
Fix computation of items' content size.
Fix update of items and scroll indicator appearance while selecting the an item from menu.

david-livadaru-3pillar committed:
feature/optional-top-menu

Update framework version.

david-livadaru-3pillar committed:
feature/optional-top-menu

Fix mispelling for appearance.
Use isValid function instead of calling the menu view controller instead.
